tests/Testcontainers.Papercut.Tests/PapercutContainerTest.cs

```diff
@@ -0,0 +1,47 @@
+using System.Diagnostics;
+using System.Linq;
+using System.Net.Mail;
+
+namespace Testcontainers.Papercut;
+
+public sealed class PapercutContainerTest : IAsyncLifetime
+{
+    private readonly PapercutContainer _papercutContainer = new PapercutBuilder().Build();
+
+    public Task InitializeAsync()
+    {
+        return _papercutContainer.StartAsync();
+    }
+
+    public Task DisposeAsync()
+    {
+        return _papercutContainer.DisposeAsync().AsTask();
+    }
+
+    [Fact]
+    [Trait(nameof(DockerCli.DockerPlatform), nameof(DockerCli.DockerPlatform.Linux))]
+    public async Task SendingAnEmail()
+    {
+        //Given
+        var client = new SmtpClient(_papercutContainer.Hostname,_papercutContainer.GetMappedPublicPort(25));
```

Comment:
WDYT about a delegate to access the SmtpPort?

Reply:
I agree this is better so have added it in